packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 24.2.0
```

Comment:
This could be a breaking change couldn't it? I know that the old behavior was essentially just "wrong", but this could "break" what used to be unintentionally correct

Reply:
The old behavior had undefined ordering, and one of the possible orderings was the one that this will guarantee. So any code that reliably behaved correctly still will, because this is one of the orderings it must have handled.